instrumentation/android-log/README.md

```diff
@@ -18,3 +18,28 @@ This instrumentation produces the following telemetry:
     * `android.log.tag`: The tag passed to `android.uti.Log.x`
     * `exception.stacktrace` - Optional. See the ([semconv here](https://github.com/open-telemetry/semantic-conventions/blob/727700406f9e6cc3f4e4680a81c4c28f2eb71569/docs/attributes-registry/exception.md#exception-stacktrace)).
     * `exception.type` - Optional. See the ([semconv here](https://github.com/open-telemetry/semantic-conventions/blob/727700406f9e6cc3f4e4680a81c4c28f2eb71569/docs/attributes-registry/exception.md#exception-type))
+
+## Installation
+
+### Applying the Byte Buddy plugin
+
+This instrumentation makes use of [Byte Buddy](https://bytebuddy.net/) to do bytecode instrumentation. Because of this, you need
+to ensure that your application has the Byte Buddy plugin applied, as shown below.
+
+```kotlin
+plugins {
+  id("net.bytebuddy.byte-buddy-gradle-plugin") version "LATEST_VERSION" // <1>
```

Comment:
We have build automation that updates the version in settings.gradle and also updates the version in the topmost `README.md` when we [preapare the release branch](https://github.com/open-telemetry/opentelemetry-android/blob/main/.github/workflows/prepare-release-branch.yml#L108). I think it would be awesome if we could expand that to cover these READMEs as well, because then they become more copy/pastable for users.

I'll open an issue.